src/main/webapp/WEB-INF/static/js/browseGermplasmListTable.js

```diff
@@ -23,75 +23,18 @@ var germplasmDataTable = null,
 
 function resetGermplasmList() {
```

Comment:
Remove resetGermplasmList, showChangeEntryTypeModal and showChangeEntryTypeModal from this file. Directly  call Angular methods using ng-click.